zerver/logging_handlers.py

```diff
@@ -16,6 +16,13 @@
 from zerver.lib.logging_util import find_log_caller_module
 from zerver.lib.queue import queue_json_publish
 
+USER_ROLE = {
+    100: 'ROLE_REALM_OWNER',
```

Comment:
Should I let this as it is, or it would be better to write "Realm Owner" instead of "ROLE_REALM_OWNER" for example, as this information will be sent in an email?

Reply:
I think it'd be better to write "Realm owner".  Ideally, we'd just use an existing mapping in zerver/models.py rather than adding a duplicate.  Let's extract `role_id_to_name_map` from `corporate/views.py` and reuse that.